src/tests/integration/D3MCompound.t.sol

```diff
@@ -96,6 +96,9 @@ contract D3MCompoundTest is DSSTest {
     uint256 constant INTEREST_RATE_TOLERANCE = WAD / 10000;
 
     function setUp() public override {
+        emit log_named_uint("block", block.number);
+        emit log_named_uint("timestamp", block.timestamp);
```

Comment:
I'm assuming these were left in by mistake?

Reply:
We left them on purpose, so if by any chance we got a failing test we could have the data available to reproduce. However it is true that the normal behavior of `forge` script makes it a bit annoying as always is being printed.
Not sure if we still want to keep it though.